style.css

```diff
@@ -549,6 +550,69 @@ body > footer {
 	}
 }
 
+@media (max-width: 1000px){
+	body {
+		grid-template:
+			'header header' auto
+			'curve preview' auto
+			'curve library' auto / 300px 1fr;
+	}
+
+	header {
+		text-align: center;
+	}
+
+	header > h1 {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: center;
+		align-items: center;
+	}
+}
+
+@media (max-width: 850px){
+	body {
+		grid-template:
+			'footer' auto
+			'header' auto
+			'curve' auto
+			'preview' auto
+			'library' auto / auto;
+		padding: 0 5% 1em;
+	}
+
+	body > footer {
+		padding: .5em 0;
+		position: static;
+		max-width: 100%;
+		transform: none;
+		white-space: normal;
+	}
+
+	h1 > a {
+		font-size: 0.8em;
+		width: 100%;
+	}
+
+	#copybuttons {
+		margin-left: auto;
+	}
+}
+
+@media (max-width: 600px){
+	h1 > a {
+		font-size: 0.7em;
```

Comment:
There's a lot of fine grained adjustment of font sizes here on what looks like it's trying to approximate a continuous adjustment that depends on viewport width. Could some expression involving `vw` units (e.g. something like `calc(0.6em + 2vw)` or something) and `clamp()` be better than multiple MQs? Also, it looks some of these rules do not have base styles (i.e. outside of any MQ), which is an antipattern. E.g. I couldn't find a `h1 > a` rule outside any MQ.